pymysqlreplication/event.py

```diff
@@ -880,12 +880,11 @@ def __init__(self, from_packet, event_size, table_map, ctl_connection, **kwargs)
         super(RowsQueryLogEvent, self).__init__(
             from_packet, event_size, table_map, ctl_connection, **kwargs
         )
-        self.query_length = self.packet.read_uint8()
-        self.query = self.packet.read(self.query_length).decode("utf-8")
+        self.packet.advance(1)
```

Comment:
How about delete line#875

Reply:
My bad. I'll fix this right away. Thx for the thorough review!😁